packages/@aws-cdk/app-delivery/lib/pipeline-deploy-stack-action.ts

```diff
@@ -1,5 +1,6 @@
 import cfn = require('@aws-cdk/aws-cloudformation');
-import codepipeline = require('@aws-cdk/aws-codepipeline-api');
+import cpactions = require('@aws-cdk/aws-codepipeline-actions');
+import cpapi = require('@aws-cdk/aws-codepipeline-api');
```

Comment:
I think it makes sense to deprecate `codepieline-api` now. We can just put these types in `codepipeline` because `-actions` takes a dependency on it.

Reply:
Done.